e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/parties/bond-beneficiary.spec.js

```diff
@@ -38,14 +38,178 @@ context('Bond beneficiary URN - User can add, edit, confirm and submit URN to th
+        pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
+
+        pages.bondBeneficiaryPage.urnInput(1).clear().type('12');
+        pages.bondBeneficiaryPage.urnInput(2).clear().type('12');
+        pages.bondBeneficiaryPage.saveButton().click();
+
+        cy.url().should('eq', relative(`/case/${dealId}/parties/${party}`));
+        pages.bondBeneficiaryPage.errorSummary().contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
+
+        pages.bondBeneficiaryPage.urnInput(1).clear().type('ABC123');
+        pages.bondBeneficiaryPage.urnInput(2).clear().type('ABC123');
+        pages.bondBeneficiaryPage.saveButton().click();
+
+        cy.url().should('eq', relative(`/case/${dealId}/parties/${party}`));
+        pages.bondBeneficiaryPage.errorSummary().contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
+
+        pages.bondBeneficiaryPage.urnInput(1).clear().type('"!£!"£!"£!"£');
+        pages.bondBeneficiaryPage.urnInput(2).clear().type('"!£!"£!"£!"£');
+        pages.bondBeneficiaryPage.saveButton().click();
+
+        cy.url().should('eq', relative(`/case/${dealId}/parties/${party}`));
+        pages.bondBeneficiaryPage.errorSummary().contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
+
+        pages.bondBeneficiaryPage.urnInput(1).clear().type('1234!');
+        pages.bondBeneficiaryPage.urnInput(2).clear().type('1234!');
+        pages.bondBeneficiaryPage.saveButton().click();
+
+        cy.url().should('eq', relative(`/case/${dealId}/parties/${party}`));
+        pages.bondBeneficiaryPage.errorSummary().contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
+
+        pages.bondBeneficiaryPage.urnInput(1).clear().type(' ');
+        pages.bondBeneficiaryPage.urnInput(2).clear().type(' ');
+        pages.bondBeneficiaryPage.saveButton().click();
+
+        cy.url().should('eq', relative(`/case/${dealId}/parties/${party}`));
+        pages.bondBeneficiaryPage.errorSummary().contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(1).contains('Enter a minimum of 3 numbers');
+        pages.bondBeneficiaryPage.urnError(2).contains('Enter a minimum of 3 numbers');
+      });
```

Comment:
Not a blocker - would be nice to split these up into their own "it's" for easier debugging; Could also create a new command to save repetition e.g `submitAndAssertValidationError(field, message)`